src/Bicep.Core.UnitTests/IServiceCollectionExtensions.cs

```diff
@@ -41,6 +43,7 @@ public static class IServiceCollectionExtensions
         .AddSingleton<IArtifactRegistryProvider, DefaultArtifactRegistryProvider>()
         .AddSingleton<ITokenCredentialFactory, TokenCredentialFactory>()
         .AddSingleton<IFileResolver, FileResolver>()
+        .AddSingleton<IEnvironment, Core.Utils.Environment>()
```

Comment:
Why are we using the actual `Environment` here in unit tests?

Reply:
Good point, fixed!